config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "rack/attack"
+
+if ENV["POD_IP"].present?
+  Rack::Attack.safelist("mark any authenticated access safe") do |request|
+    Rails.logger.warn("Safelisted: #{request.ip} #{request.path}")
+    request.ip == ENV["POD_IP"] && req.path.start_with?("/health_check")
+  end
+end
+
+Rack::Attack.enabled = ActiveRecord::Type::Boolean.new.cast(ENV.fetch("RACK_ATTACK_ENABLED", "true"))
+
+if Rails.env.production?
+  class Rack::Attack
+    throttle("req/ip", limit: Decidim.throttling_max_requests, period: Decidim.throttling_max_requests) do |req|
+      Rails.logger.warn("[Rack::Attack] [THROTTLE - req / ip] :: #{req.ip} :: #{req.path} :: #{req.GET}")
```

Comment:
This one isn't efficient, I made a mistake before, prefer to use initializer from [decidim-app](https://github.com/OpenSourcePolitics/decidim-app/blob/develop/config/initializers/rack_attack.rb)